### ISSUE TITLE
Speed up tool processing by turning off map layers

### ISSUE THREAD

**Issue by RichThomasUniti** [uniti-tim/gis_multi_site#24]
Joel,
I found this solution that would speed up processing, since it's hard for users to remember to pause drawing the map Data View before executing the tool.
https://gis.stackexchange.com/questions/1644/how-to-disable-and-enable-drawing-in-arcmap-using-python
DO you think this would work well?

**Comment by joelherr** [comment]
I think the bottom option on that link is the way to go.  Should be pretty straightforward to turn all the layers off before the script starts running and then turn them back on when it's done.

**Comment by RichThomasUniti** [comment]
Exactly. I forgot there were multiple options suggested on that question, but it was the bottom option you saw that I thought would be best.
